test/NetTopologySuite.Tests.NUnit/LinearReferencing/LengthLocationMapTests.cs

```diff
@@ -0,0 +1,259 @@
+﻿using System;
+using NetTopologySuite.Geometries;
+using NetTopologySuite.IO;
+using NetTopologySuite.LinearReferencing;
+using NUnit.Framework;
+
+namespace NetTopologySuite.Tests.NUnit.LinearReferencing
+{
+    [TestFixture]
+    public class LengthLocationMapTests
```

Comment:
Derive from GeometryTestCase, this gives you access to `Read` wkt functionality and more